lib/semian/circuit_breaker.rb

```diff
@@ -116,5 +118,9 @@ def log_state_transition(new_state)
       str << " error_timeout=#{@error_timeout} error_last_at=\"#{@errors.last}\""
       Semian.logger.info(str)
     end
+
+    def disabled?
+      ENV['SEMIAN_CIRCUIT_BREAKER_DISABLED'] || ENV['DISABLE_SEMIAN']
```

Comment:
Nitpick, but I'd prefer if all environment variables used the same prefix,`SEMIAN_` (rename `DISABLE_SEMIAN` to `SEMIAN_DISABLED`).